### DISCUSSION TITLE
XCalendar .net Maui latest version with .net framework 7.0 works when debugging, crashes at startup when installed on android phone

### DISCUSSION THREAD

**Discussion by EugenM88** [ME-MarvinE/XCalendar#135] [Q&A]
Greetings!

After successfully implementing this amazing control and trying to test it on my phone (Realme 8 Pro with Android 13)
The program crashes at startup after installation.

The program works just fine in the emulator when debugging, but when changing to release and trying to manually install the app into my android phone, the program installs fine, however when starting the program up, the .net maui splashscreen appears and then app instantly crashes without any error.

I have tried using a clean project from scratch, same problem persists.
Is there any tip or anything i could do to make it run on a android phone?

Thanks in advance for all your help!

**Comment by ME-MarvinE** [comment]
This sounds like an issue, can you create an issue for this in the repo? The information would be useful and I'll have some questions to ask ones its created. It may be related to #67.

For version numbers, in some cases people don't realise they're on a different or outdated version so it always helps to explicitly state it in order to avoid any confusion.

**Comment by EugenM88** [comment]
Hi there! Thank you, as per your suggestion i have created a new issue:
[#136](https://github.com/ME-MarvinE/XCalendar/issues/136)

Best Regards!
